### DISCUSSION TITLE
ipal sources?

### DISCUSSION THREAD

**Discussion by hpax** [steveicarus/iverilog#1109] [General]
Hi,

I would like to take a stab at revive the tgt-pal backend, but I cannot find the ipal.h file, which I understand is part of a separate library but which is/was part of the overall iverilog project. Would it be possible to post those sources in a git repository?

Thanks.

**Comment by steveicarus** [comment]
I don't even remember where they were" part of iverilog or separate. I'll
take a look around.


On Sun, Mar 10, 2024 at 3:28 PM H. Peter Anvin ***@***.***>
wrote:

> Hi,
>
> I would like to take a stab at revive the tgt-pal backend, but I cannot
> find the ipal.h file, which I understand is part of a separate library but
> which is/was part of the overall iverilog project. Would it be possible to
> post those sources in a git repository?
>
> Thanks.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/steveicarus/iverilog/discussions/1109>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAACKMU2YKL57CUXOLESSPTYXTUBLAVCNFSM6AAAAABEPJ75LOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGM2TIMJSGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Steve Williams
***@***.*** ***@***.***>

**Comment by martinwhitaker** [comment]
The Wayback Machine captured the project page (https://web.archive.org/web/20220831214611/http://icarus.com/eda/ipal/) but not the source files.
